src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -287,6 +287,9 @@ const InstanceSnapshots: FC<Props> = ({ instance }) => {
             allNames={
               instance.snapshots?.map((snapshot) => snapshot.name) ?? []
             }
+            onUpdateSort={pagination.updateSort}
+            defaultSort="created_at"
+            defaultSortDirection="descending"
```

Comment:
For small screens, this is currently working only when you open the Snapshots tab. Then, as soon as you alter the sorting of any column, there's no way to "restore" this default way of sorting the data. It will be either sorted by name - even though the "arrow" is placed next to "Date created", which is confusing - or, if you "reset" the sort status of this column (= no arrow visible anymore), it will be sorted by date created but ascending instead of descending:

![Peek 2023-04-14 10-50](https://user-images.githubusercontent.com/56583786/231995890-95601703-9655-407d-9f2f-943b5899ce5a.gif)

Reply:
This is a little tricky to solve. Some options that come to mind:
* Sort the collapsed column by date created instead of name
* Change which columns are collapsed, and not allow sorting by the collapsed column
* Disallow sorting the collapsed column as it is

Reply:
I applied the 1. Idea and think it is actually pretty neat.